### PR TITLE
[FEATURE] Order factory

### PR DESCRIPTION
**Pull request by turbo-ele**

### CODE REVIEW

File: Api/ApiOrderInterface.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\Sales\OrderBundle\Api;
+
+interface ApiOrderInterface
+{
+    // TODO: implement
```
Comment (christianbader) [inline Api/ApiOrderInterface.php:15]: do it

File: Cart/CartManager.php [hunk 2]
```diff
@@ -103,11 +101,23 @@ class CartManager extends BaseSalesManager
...
+    protected $orderFactory;
+
+    /**
      * @param ObjectManager $em
      * @param SessionInterface $session
      * @param OrderRepository $orderRepository
      * @param OrderManager $orderManager
      * @param GroupedItemsPriceCalculatorInterface $priceCalculation
+     * @param string $defaultCurrency
+     * @param object $accountManager
```
Comment (christianbader) [inline Cart/CartManager.php]: what kind of object?

File: Cart/CartManager.php [hunk 3]
```diff
@@ -120,7 +130,8 @@ public function __construct(
         \Twig_Environment $twig,
         OrderPdfManager $pdfManager,
         \Swift_Mailer $mailer,
-        $mailerFrom
+        $mailerFrom,
+        OrderFactoryInterface $orderFactory
     )
     {
```
Comment (christianbader) [inline Cart/CartManager.php]: move up

**Comment by christianbader** [comment]
else ack

File: Api/ApiOrderInterface.php (hunk 1 above)
Comment (turbo-ele) [inline Api/ApiOrderInterface.php:15]: will be done in next PR

**Pull request merged**